nextflow_schema.json

```diff
@@ -280,7 +298,7 @@
                 },
                 "annotation_prokka_evalue": {
                     "type": "number",
-                    "default": 0.000001,
+                    "default": 1e-6,
```

Comment:
```suggestion
                    "default": 0.000001,
```

Reply:
why not have it in scientific format?

Reply:
Because prokka doesn't accept it. Updating `schema.json` in the web builder might change certain values for whatever reason. Also make sure to use the most recent version of tools, to minimize these (almost) silent changes.

Reply:
This is not committed yet.